index.html

```diff
@@ -380,6 +380,42 @@ <h3>Terminal</h3>
             </ul>
         </section>
 
+        <section class="form-section">
+
+            <div class="form-intro-container">
+                <p>
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </p>
+            </div>
+            <form action="https://formspree.io/f/mnqynkjg" method="POST">
+                <div class="name-div">
+                    <input type="text" name="name" placeholder="Full name" required>
+                    <hr>
+                </div>
+                <div class="email-div">
+                    <input type="email" name="email" placeholder="Email address" required>
+                    <hr>
+                </div>
+                <div class="message-div">
+                    <textarea name="message" rows="8" cols="39" maxlength="200" required>Enter text here</textarea>
```

Comment:
Your form looks great and submission works. Kindly include the maxlength validations as specified in the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/096130a7c3e71e15c3c97ddc2ed0ec6467621a8c/portfolio/3_contact_form.md?plain=1#L41).